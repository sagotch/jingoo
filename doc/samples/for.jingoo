{%- for item in [ 0, 1, 2, 3, 4, 5 ] -%}
  - {{ item }}: {{ loop.cycle ("even","odd") }}
{% endfor -%}
