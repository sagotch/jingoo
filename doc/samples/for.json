{ "links" : [ [ "http://yahoo.co.jp", "yahoo japan" ],
	      [ "http://google.co.jp", "google japan" ] ]
  , "obj": { "name": "alice", "age": 42, "sex": "F" }
}
