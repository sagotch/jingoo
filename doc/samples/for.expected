- 0: even
- 1: odd
- 2: even
- 3: odd
- 4: even
- 5: odd
